### PR TITLE
Add Serverless Aurora post

### PR DESCRIPTION
**Pull request by alexdebrie**

### CODE REVIEW

**Comment by andrea-p** [comment]
schedule(12/04/2017 6:00am)

**Comment by DavidWells** [comment]
✅ Post scheduled for **12/04/2017 at 6:00amundefined (America/Los_Angeles)**

**To unscheduled the post:**

1. **[Delete the previous comment ⬆](https://github.com/serverless/blog/pull/442#issuecomment-348872940)** with schedule(time) information by clicking the ❌
  ```
  ┌──────────────────────────────────────────────┐
  │                                           X  │
  ├───────────────────────────────────────────▲──┤
  │                                           │  │
  │  schedule(mm/dd/yyyy h:mm A)   ┌────────┐ │  │
  │                                │ Delete ├─┘  │
  │                                └────────┘    │
  └──────────────────────────────────────────────┘
  ```
2. (Optionally) Delete this comment to keep things tidy

> Powered by the ⚡️ [Serverless post scheduler](https://github.com/serverless/post-scheduler) | Made with ❤️  by [Serverless](https://serverless.com/)

File: posts/2017-12-04-serverless-aurora-future-of-data.md [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+
+### Server-full databases
+
+A server-full approach uses instance-based solutions such as MySQL, Postgres, or MongoDB. I classify them as instance-based when you can tell me how many instances you have running and what their hostnames are.
+
+I like Postgres + Mongo because of [their popularity](https://db-engines.com/en/ranking), which means data design patterns are well-known and language libraries are mature.
+
+However, these instance-based solutions were designed for a pre-serverless world with long-running compute instances. This leads to the following problems:
+
+_Connection Limits_
```
Comment (mthenw) [inline posts/2017-12-04-serverless-aurora-future-of-data.md]: IMHO the biggest problem for Lambda was establishing connection phase. It took some time to connect and it influenced the coldstart. The second issue that affected coldstart even more was that lambda had to run in VPC to connect to DB.

Comment (mthenw) [reply posts/2017-12-04-serverless-aurora-future-of-data.md]: In general I agree with everything you wrote in this section.

**Comment by mthenw** [comment]
@alexdebrie great post!

**Comment by DavidWells** [comment]
🎉 **The post has been published**

> Powered by the ⚡️ [Serverless post scheduler](https://github.com/serverless/post-scheduler) | Made with ❤️  by [Serverless](https://serverless.com/)

**Pull request merged**